src/include/daos_task.h

```diff
@@ -445,6 +446,7 @@ typedef struct {
 	unsigned int		nr;
 	daos_iod_t		*iods;
 	d_sg_list_t		*sgls;
+	struct dcs_iod_csums	*iod_csums;
```

Comment:
this is still a public struct so why would you add checksums here?
is there another way to pass the checksums internally other than the task args?

Reply:
I didn't realize the task stuff was public ... I will move it to a more appropriate place. Thinking about the obj/shard_auxi_args??